source/client-side-encryption/client-side-encryption.rst

```diff
@@ -10,8 +10,8 @@ Client Side Encryption
 :Status: Accepted
 :Type: Standards
 :Minimum Server Version: 4.2 (CSFLE), 6.0 (Queryable Encryption)
-:Last Modified: 2022-06-23
-:Version: 1.8.1
+:Last Modified: 2022-06-27
+:Version: 1.9.0
```

Comment:
No objections to a minor bump here (per your explanation in the OP), but that's more because I personally don't think these versions serve much purpose.